app/src/main/java/io/appium/uiautomator2/utils/AXWindowHelpers.java

```diff
@@ -99,20 +104,46 @@ private static AccessibilityNodeInfo[] getWindowRoots() {
         return result.toArray(new AccessibilityNodeInfo[0]);
     }
 
+    private static AccessibilityNodeInfo getTopmostWindowRootFromActivePackage() {
+        CharSequence activeRootPackageName = getActiveWindowRoot().getPackageName();
```

Comment:
I presumed so since it is guaranteed that `getActiveWindowRoot()` method returns a non-null `AccessibilityNodeInfo` and [getPackageName()](https://android.googlesource.com/platform/frameworks/base/+/master/core/java/android/view/accessibility/AccessibilityNodeInfo.java#2909) of the instance return type is `CharSequence `.

Is it preferable to add null check here?

Reply:
I think it makes sense to have it. The code would be safer that way.

Reply:
Basically it would be enough to just have checkNotNull assertion

Reply:
Thanks for suggestion. Do you imply the change like this one?

```java
import androidx.test.internal.util.Checks.checkNotNull;

...
    private static AccessibilityNodeInfo getTopmostWindowRootFromActivePackage() {
        CharSequence activeRootPackageName = getActiveWindowRoot().getPackageName();
        checkNotNull(activeRootPackageName);
```

Reply:
checkNotNull returns the same value, so it could look like `CharSequence activeRootPackageName = checkNotNull(getActiveWindowRoot().getPackageName())`

Reply:
Addressed in 8d5012e8a16b953288efc429a2e3a02a06176884. Does this import statement's position look right?